src/lsm/groove.zig

```diff
@@ -777,24 +778,26 @@ pub fn GrooveType(
             }
 
             fn lookup_with_timestamp(worker: *PrefetchWorker, timestamp: u64) void {
-                if (worker.context.groove.objects.lookup_from_memory(
+                switch (worker.context.groove.objects.lookup_from_memory(
                     worker.context.snapshot,
                     timestamp,
-                )) |object| {
-                    // The object is not a tombstone; the ID (if any) and Object trees are in sync.
-                    assert(!ObjectTreeHelpers(Object).tombstone(object));
-
-                    worker.context.groove.prefetch_objects.putAssumeCapacityNoClobber(object.*, {});
-                    worker.lookup_start_next();
-                    return;
+                )) {
+                    .exists => |value| {
```

Comment:
Be explicit -- capture by pointer & pass it to lookup_object_callback.

Reply:
We aren't taking a copy here, `lookup_from_memory` returns a `?*const Value`, so we aren't taking a copy here.